event-stream-processing/src/constants/fingerprints.ts

```diff
@@ -166,6 +166,35 @@ export const FINGERPRINTS: OsDocumentFingerprint[] = [
       },
     },
   },
+  {
+    name: FingerprintCategory.TOMCAT_LOG4J_LOGS,
+    fingerprint: {
+      event: {
+        kind: 'event',
+        category: 'web',
```

Comment:
Category should be an array.

https://www.elastic.co/guide/en/ecs/current/ecs-event.html#field-event-category

Reply:
Such as category: ['web']?

Reply:
@mbystedt I updated all the event categories accordingly.